src/DynamoCore/Graph/Nodes/CodeBlockNode.cs

```diff
@@ -298,23 +299,12 @@ public void SetCodeContent(string newCode, ElementResolver workspaceElementResol
             }
         }
 
-        /// <summary>
-        /// Temporary variables that generated in code.
-        /// </summary>
-        [JsonIgnore]
-        public IEnumerable<string> TempVariables
-        {
-            get { return tempVariables; }
-        }
```

Comment:
This list was never used/modified. Deleting...

Reply:
If this list did not have any values added to it, were its usages always returning false?